types/intersects/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "@definitelytyped/dtslint/dt.json" }
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "unnecessary-bind": false
```

Comment:
Why do you need to disable this rule?

Reply:
I disabled this as I was getting this lint error when running `npm run test intersects` locally:

```
The 'unnecessary-bind' rule threw an error in '/home/clemens/Repos/DefinitelyTyped/types/intersects/circleOutline-circle.d.ts':
TypeError: DeprecationError: 'originalKeywordKind' has been deprecated since v5.0.0 and can no longer be used. Use 'identifierToKeywordKind(identifier)' instead.
```

Interestingly running `rm -rf node_modules && npm install` made the error go away. Now I got a different issue:

```
/home/clemens/Repos/DefinitelyTyped/types/intersects/polygon-polygon.d.ts
  1:1  error  Definition for rule 'dt-header' was not found                           dt-header
  1:1  error  Definition for rule 'export-just-namespace' was not found               export-just-namespace
  ...
```

Switching explicitly to node 16, rebasing the branch to the latest version, and re-installing the modules made the error go away, so I removed the override in 2b4ac8dfa.